test/controllers/ReturnSentControllerSpec.scala

```diff
@@ -17,30 +17,550 @@
 package controllers
 
 import base.SpecBase
+import connectors.SoftDrinksIndustryLevyConnector
+import models.retrieved.RetrievedActivity
+import models.{Amounts, ReturnPeriod, SmallProducer, UserAnswers}
+import org.jsoup.Jsoup
+import org.mockito.ArgumentMatchers.any
+import org.mockito.Mockito.when
+import org.mockito.MockitoSugar.mock
+import org.scalatest.BeforeAndAfter
+import play.api.i18n.Messages
+import play.api.inject.bind
+import play.api.libs.json.Json
 import play.api.test.FakeRequest
 import play.api.test.Helpers._
-import views.html.IndexView //TODO will change to return sent view
+import repositories.SDILSessionCache
+import viewmodels.govuk.SummaryListFluency
 
-class ReturnSentControllerSpec extends SpecBase {
+import scala.concurrent.Future
+
+class ReturnSentControllerSpec extends SpecBase with SummaryListFluency with BeforeAndAfter {
 
   "ReturnSent Controller" - {
 
-    "must show returns sent page" in {
+    val zero = BigDecimal(0.00)
+    val amounts = Amounts(zero, zero, zero)
+    val mockSessionCache = mock[SDILSessionCache]
+    when(mockSessionCache.fetchEntry[Amounts](any(), any())(any())) thenReturn Future.successful(Some(amounts))
+    val returnPeriodsContainingBaseReturnPeriod = List(ReturnPeriod(2020, 0), ReturnPeriod(2023, 1), returnPeriod)
```

Comment:
This is not used, can be removed. You can also remove ReturnPeriod from the imports.